src/Nethermind/Nethermind.Blockchain/KnownAddresses.cs

```diff
@@ -27,15 +27,15 @@ public static string GetDescription(Address address)
             {
                 return GoerliValidators[address];
             }
-            
+
             if (RinkebyValidators.ContainsKey(address))
             {
                 return RinkebyValidators[address];
             }
 
             return "?";
         }
-        
+
         public static Dictionary<Address, string> GoerliValidators = new Dictionary<Address, string>
```

Comment:
Maybe in future load it from config instead of hardcode?

Reply:
this is not that important - more like a fun feature, any additional config is a maintenance burden and raise questions by users which is not needed here